src/pipeline/rackhd/ova/Build.groovy

```diff
@@ -0,0 +1,68 @@
+package pipeline.rackhd.ova
+
+def execute(){
+  def shareMethod = new pipeline.common.ShareMethod()
+
+  def label_name = "packer_ova"
+  lock(label: label_name, quantity: 1){
+    node_name = shareMethod.occupyAvailableLockedResource(label_name, [])
+    node(node_name){
+      ws{
+          withEnv([
+              "RACKHD_COMMIT=${env.RACKHD_COMMIT}",
+              "RACKHD_VERSION=${env.RACKHD_VERSION}",
+              "IS_OFFICIAL_RELEASE=${env.IS_OFFICIAL_RELEASE}",
+              "OVA_CACHE_BUILD=${env.OVA_CACHE_BUILD}",
+              "OS_VER=${env.OS_VER}",
+              "BUILD_TYPE=vmware",
+              "ARTIFACTORY_URL=${env.ARTIFACTORY_URL}",
+              "STAGE_REPO_NAME=${env.STAGE_REPO_NAME}",
+              "DEB_COMPONENT=${env.DEB_COMPONENT}",
+              "DEB_DISTRIBUTION=trusty"]) {
```

Comment:
Please use them as arguments of the function.
Or handle it in your next PR.

Reply:
I'm writing next PR